examples/python-kernel-wrapper/cython/src/kernel.cu

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+#include <cudf/column/column_device_view.cuh>
+#include <cudf/table/table_device_view.cuh>
+
+static const float mm_to_inches = 0.0393701;
+
+__global__ void kernel_tenth_mm_to_inches(cudf::mutable_column_device_view column, int64_t num_rows)
+{
+    int i = blockIdx.x * blockDim.x + threadIdx.x;
+    if (i < num_rows) {
+        column.element<int64_t>(i) = column.element<int64_t>(i) * (1/10) * mm_to_inches;
+    }
+}
```

Comment:
Couple of things. 
 - Use constexpr
 - size of columns is `cudf::size_type` which is 32-bit. But you don't need that parameter anyway, since it's part of the device_view.

```suggestion
static constexpr float mm_to_inches = 0.0393701;

__global__ void kernel_tenth_mm_to_inches(cudf::mutable_column_device_view column)
{
    int i = blockIdx.x * blockDim.x + threadIdx.x;
    if (i < column.size()) {
        column.element<int64_t>(i) = column.element<int64_t>(i) * (1/10) * mm_to_inches;
    }
}
```

Technically this should also use a grid-stride loop rather than an if() because if we every change size() to be a 64-bit value, it will be possible to have more rows than you can process with a single 1D CUDA grid.

To be honest, best practice would be to use `thrust::transform` with a lambda to implement this rather than a kernel...